cloudferrylib/os/actions/filter_similar_vms_from_dst.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2015: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import collections
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils
+
+LOG = utils.get_log(__name__)
+
+COMPARISON_FIELDS = ['name', 'flav_details', 'key_name', 'interfaces',
+                     'volumes']
+
+
+class FilterSimilarVMsFromDST(action.Action):
```

Comment:
Please add docstring with:
 - General action overview
 - Required config options
 - Dependencies and dependents